Core/HLE/proAdhocServer.cpp

```diff
@@ -1376,8 +1376,8 @@ void game_product_override(SceNetAdhocctlProductCode * product)
 				// Destroy Prepare SQL Statement
 				sqlite3_finalize(statement);
 			}*/
-			for (std::vector<db_productid>::iterator it = productids.begin(); it != productids.end(); it++) {
-				if (IsMatch(it->id, productid)) {
+			for (const auto &product: productids) {
```

Comment:
nit:
```suggestion
			for (const auto &product : productids) {
```

-[Unknown]